### PR TITLE
Staggered Release Endpoints part 6

### PR DESCRIPTION
**Pull request by brendanshean**
## WHAT
Call `save_user_pack_sequence_items` for relevant objects that update without triggering callbacks.

Also, remove a rake task and it's associated worker since classroom activities aren't used anymore and the worker has several `update_all` calls that would in turn require some analysis user_pack_sequence_items:
lib/tasks/classroom_activities_to_unit_activities_and_classroom_units.rake
app/workers/classroom_activities_to_unit_activities_and_classroom_units_worker.rb

## WHY
`update_all` or `save(validate: false)` do not trigger callbacks which means that an objects associated user_pack_sequence_item status can become stale.

## HOW
Add `save_user_pack_sequence_items` in places following updates that do not trigger callbacks.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | N/A

### CODE REVIEW

File: services/QuillLMS/app/controllers/concerns/teacher_fixes.rb [hunk 1]
```diff
@@ -105,49 +105,49 @@ def self.merge_two_schools(from_school_id, to_school_id)
     SchoolsUsers.where(school_id: from_school_id).update_all(school_id: to_school_id)
   end
 
-  def self.merge_two_classrooms(class_id1, class_id2)
-    move_students_from_one_class_to_another(class_id1, class_id2)
+  def self.merge_two_classrooms(classroom_id1, classroom_id2)
```
Comment (dandrabik) [inline services/QuillLMS/app/controllers/concerns/teacher_fixes.rb]: Take or leave. If we are renaming these, we might consider something like `source_classroom`, `target_classroom` or something like that.

File: services/QuillLMS/app/controllers/teachers/unit_activities_controller.rb [hunk 2]
```diff
@@ -39,12 +40,10 @@ def update_multiple_dates
...
   end
 
-  private def set_activity_session
+  private def set_activity_sessions
     activity_ids = @unit_activities.map(&:activity_id).flatten.uniq
     classroom_unit_ids = []
-    @unit_activities.each do |ua|
-      classroom_unit_ids << ua.unit.classroom_unit_ids
-    end
+    @unit_activities.each { |ua| classroom_unit_ids << ua.unit.classroom_unit_ids }
```
Comment (dandrabik) [inline services/QuillLMS/app/controllers/teachers/unit_activities_controller.rb]: Consider
```ruby
classroom_unit_ids = @unit_activities.map { |ua| ua.unit.classroom_unit_ids }.flatten.uniq

**Review by dandrabik — APPROVED** [review]
Looks good to me. Small comments on things you refactored and not your added code (so feel free to ignore).

**Review by emilia-friedberg — APPROVED** [review]
LGTM! Thanks for that cleanup.

**Pull request merged**